extensions/sql-migration/src/telemtery.ts

```diff
@@ -4,9 +4,12 @@
  *--------------------------------------------------------------------------------------------*/
 
 import AdsTelemetryReporter, { TelemetryEventMeasures, TelemetryEventProperties } from '@microsoft/ads-extension-telemetry';
-import { getPackageInfo } from './api/utils';
 const packageJson = require('../package.json');
-let packageInfo = getPackageInfo(packageJson)!;
+let packageInfo = {
```

Comment:
Had to remove `getPackageInfo()` from utils and replace it inline here, otherwise there would be a circular dependency issue since `utils.ts` now needs to import this file (in order to properly emit errors in utility methods to telemetry) whereas this file would've needed to import `utils.ts`